dev/format_code.sh

```diff
@@ -18,6 +18,7 @@ set -euo pipefail
 
 DOCKER=""
 DOCKER_CMD="docker"
+CLANGFORMAT=${CLANGFORMAT-clang-format}
```

Comment:
That syntax looks weird. Shouldn't this be `${CLANGFORMAT:clang-format}`?

Reply:
I had found it after a google search. After more research it might be `${CLANGFORMAT:-clang-format}`
https://wiki.bash-hackers.org/syntax/pe#use_a_default_value
Will overwrite the unset and the variables set to null

Reply:
Ah, yeah, that's what I meant to write above 🙂 Can you update your pull request?

Reply:
@dbrgn now it should be fine, right?

Reply:
I think so, yes!